src/types/onyx/ExpensifyCard.ts

```diff
@@ -0,0 +1,26 @@
+import type * as OnyxCommon from './OnyxCommon';
+import type PersonalDetails from './PersonalDetails';
+
+/** Model of an Expensify card */
+type ExpensifyCard = OnyxCommon.OnyxValueWithOfflineFeedback<{
```

Comment:
We already have Card.ts

https://github.com/Expensify/App/blob/3622deb409a4cf91a47cc9196e2414ffe75f0f16/src/types/onyx/Card.ts

Let's reuse this one instead of creating a new model

Reply:
Card.ts currently does not have the accountID but we have have the backend return that. Can we get personalDetails via the accountID in the App? Or would we rather have the backend return the personal details?

Reply:
@MariaHCD accountID - of the cardholder? yeah, we can get personal details with accountID on the FE

Reply:
Yes, accountID of the cardholder 👍🏼

Reply:
Are we planning to remove this model?

Reply:
+1, I assume we could reuse the same model